.expeditor/scripts/release_habitat/shared.sh

```diff
@@ -22,13 +26,20 @@ import_keys() {
         core
 }
 
-get_latest_pkg_version_in_channel() {
+get_latest_pkg_release_in_channel() {
```

Comment:
Yeah these two names are not great, the idea is to get the full release name `0.XX.Y/0339494949` vs `0.XX.Y`. I don't think `get_latest_pkg_in_release_channel` makes that clearer since it's not getting a package, it's getting a version.

Reply:
Ah, yeah that makes sense.   `get_latest_(something)_in_release_channel` where `something` indicates we're getting a `version + release` would be ideal but I don't know that we have a term to describe that pair of the identifier.   Maybe a short `# Returns the 'version/release' of the requested package in the release channel` comment would be sufficient?